### PR TITLE
Update ginkgo v2.8.3

### PR DESCRIPTION
**Pull request by lyarwood**
**What this PR does / why we need it**:

Update ginkgo v2.8.3 and move `unittest` target to use `go run` as `functest` already does.

**Special notes for your reviewer**:

Based on https://github.com/kubevirt/ssp-operator/pull/495 so WIP until that lands.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,18 +83,18 @@ endif
...
 
+GINKGO_VERSION ?= v2.8.3
+GINKGO_TIMEOUT ?= 2h
+GINKGO_RUN ?= go run github.com/onsi/ginkgo/v2/ginkgo@$(GINKGO_VERSION) -v -coverprofile cover.out -timeout $(GINKGO_TIMEOUT)
+
 .PHONY: unittest
-unittest: generate lint fmt vet manifests
-	go test -v -coverprofile cover.out $(SRC_PATHS_TESTS)
-	cd api && go test -v ./...
+unittest: generate fmt vet manifests
```
Comment (0xFelix) [inline Makefile:91]: Should depend on `gingko` target too? No more linting?

Comment (lyarwood) [reply Makefile:91]: Yup, bad rebase thanks for catching that.

**Comment by lyarwood** [comment]
Duplicate of #512 that also updates various other deps, I'll create a new PR moving the `unittest` target once that lands.

**Pull request closed without merging**